environment-api/src/main/java/com/sequenceiq/environment/api/v1/environment/endpoint/EnvironmentEndpoint.java

```diff
@@ -227,4 +228,19 @@ void postStartByCrn(@ValidCrn(resource = CrnResourceDescriptor.ENVIRONMENT) @Pat
     @ApiOperation(value = EnvironmentOpDescription.UPDATE_CONFIG_BY_CRN, produces = MediaType.APPLICATION_JSON, notes = ENVIRONMENT_NOTES,
         nickname = "updateConfigsInEnvironmentByCrnV1")
     void updateConfigsInEnvironmentByCrn(@ValidCrn(resource = CrnResourceDescriptor.ENVIRONMENT) @PathParam("crn") String crn);
+
+    @POST
+    @Path("/name/{name}/update_load_balancers")
+    @Produces(MediaType.APPLICATION_JSON)
+    @ApiOperation(value = EnvironmentOpDescription.UPDATE_LOAD_BALANCERS_BY_ENV_NAME, produces = MediaType.APPLICATION_JSON, notes = ENVIRONMENT_NOTES,
+        nickname = "updateEnvironmentLoadBalancersByNameV11")
+    void updateEnvironmentLoadBalancersByName(@PathParam("name") String envName, @NotNull EnvironmentLoadBalancerUpdateRequest request);
```

Comment:
why we dont using a put endpoint for this?

Reply:
I mean the current environment service PUT endpoint

Reply:
I wasn't familiar with the PUT method, but after reading about it it does make more sense. I'll update the API call.